src/pages/ReportParticipantsPage.js

```diff
@@ -57,27 +57,25 @@ const defaultProps = {
 const getAllParticipants = (report, personalDetails) => {
     const {participants} = report;
 
-    return _.chain(participants)
-        .compact()
-        .map(participants, (login) => {
-            const userLogin = Str.removeSMSDomain(login);
-            const userPersonalDetail = lodashGet(personalDetails, login, {displayName: userLogin, avatar: ''});
+    return _.map(participants, (login) => {
+        const userLogin = Str.removeSMSDomain(login);
```

Comment:
You want to access this safely too?

Reply:
Don't think it's necessary. We can remove

Reply:
lol I just found out the participants code that we merged to `main` is kinda buggy somehow, let's just revert 😓